airflow/kubernetes/pod_launcher.py

```diff
@@ -105,13 +113,14 @@ def _monitor_pod(self, pod: Pod, get_logs: bool) -> Tuple[State, Optional[str]]:
             for line in logs:
                 self.log.info(line)
         result = None
+        while self.base_container_is_running(pod):
+            self.log.info('Container %s has state %s', pod.name, State.RUNNING)
+            time.sleep(2)
```

Comment:
Might be a good idea to extract any magic numbers to a config

Reply:
ok no problem